content/ui-patterns/saving.mdx

```diff
@@ -0,0 +1,266 @@
+    <Caption>Don't automatically save selections once the dropdown is closed</Caption>
+  </Dont>
+</DoDontContainer>
+
+## Automatic saving
+
+Automatic saving applies changes as they are made. Automatic saving should be used when the user expects instant feedback from the change they made.
+
+<Box mb={3} display="flex" alignItems="center" flexDirection="column" mt={6} mx="auto" sx={{gap: 2, maxWidth: '450px'}}>
+    <Box display="inline-block" overflow="hidden" borderWidth="1px" borderColor="border.default" borderStyle="solid" borderRadius="12px">
+        <img
+            width="456"
+            src="https://user-images.githubusercontent.com/2313998/152013139-02eb5213-bb19-4c8d-8b19-01f1d73602d1.gif"
+            alt="Opening the 'Watch' dropdown menu and changing. It saves automatically."
+        />
+    </Box>
+    <Text as="p" mt="0" align="center" fontSize={1}>If a form field is saved automatically, it should be obvious whether or not it saved without using text or visual indicators. A visual indicator for whether or not a form field has been saved successfully could be mistaken as a validation status.</Text>
+</Box>
+
+### Dropdown menu accessibility
+
+Automatically saving a single-select dropdown menu is accessible as long as it is a semantic menu. A semantic menu can only be a list of items that can be selected.
+
+As soon as a dropdown has additional features (search inputs, filters, tabs, etc), the dropdown should be treated like a dialog with a button to submit and a button to cancel. A dialog will provide the proper navigation features for folks using assistive technology.
+
+## Feedback
+
+If the change is not obvious within the user's viewport, provide feedback to let the user know that the change occurred successfully.
+
+- If the change occurred without a redirect or page refresh, you can use the toast component.
+- If the change occurred and the page refreshes or redirects, you can use a flash banner component.
+
+<Note variant="warning">If your design is being implemented in github.com, avoid using a toast for now. Those toasts have accessibility issues that need to be fixed.</Note>
+
+## Error forgiveness
+
+Before submitting a form with very dangerous consequences (for example, delete a repository), ask the to user confirm they want to submit the form.
+
+For risky changes, provide a way to undo those changes after saving.
+
+<Box display="flex" alignItems="center" flexDirection="column"mx="auto" sx={{gap: 1}}>
+    <Box display="inline-block" overflow="hidden" borderWidth="1px" borderColor="border.default" borderStyle="solid" borderRadius="12px">
+        <img width="960" src="https://user-images.githubusercontent.com/2313998/152434199-a067b037-baf8-4093-8796-b0bb58344223.png" />
+    </Box>
+    <Text as="p" mt="0" align="center" fontSize={1}>After merging a pull request, a button appears to revert the changes</Text>
+</Box>
+
+## Redirecting
```

Comment:
Do we have these patterns on a different page? Knowing I can do this is helpful, but I'm not sure what patterns I should use and have available

Reply:
We don't have specific patterns defined for this yet